core/src/main/scala/com/lightbend/paradox/ParadoxProcessor.scala

```diff
@@ -86,10 +87,12 @@ class ParadoxProcessor(reader: Reader = new Reader, writer: Writer = new Writer)
     lazy val getNavigation = writer.writeNavigation(pageToc.root(loc), context)
     lazy val hasSubheaders = page.headers.nonEmpty
     lazy val getToc = writer.writeToc(headerToc.headers(loc), context)
+    lazy val getSource_url = githubLink(Some(loc)).getHtml
```

Comment:
On the properties it's snake_case, and in Scala it's camelCase. We should avoid weird hybrid.